keyboards/handwired/orbweaver/orbweaver.c

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 a_marmot
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// This IS31FL3731 configuration is for a Razer Orbweaver Chroma.  
+// It was reverse-engineered by testing all possible Cx_y combinations (144 total), 
+// and determining which ones addressed individual diodes (60 total). These were then
+// placed in order R-G-B, sweeping left to right, and then top to bottom.  
+
+#include "config.h"
+#include QMK_KEYBOARD_H
```

Comment:
```suggestion
#include "rgb_matrix.h"
```

Reply:
I have not tested this.  Can you explain?  How can this compile without config.h and QMK_KEYBOARD.H ?

Reply:
And yet it does work...